spec/requests/api/v1/sessions_spec.rb

```diff
@@ -20,11 +20,33 @@
 
       data = JSON.parse(response.body, symbolize_names: true)[:data]
 
+      expect(User.last.api_key).to eq(data[:attributes][:api_key])
       expect(data[:attributes][:username]).to eq(user.username)
     end
   end
 
   describe 'sad' do
+    it 'should not find user not there' do
```

Comment:
This is good, although we will need more sad path tests like:

- Incorrect password
- mismatch on password and confirmation
- Empty username

Reply:
Agreed. The push so far was mostly a proof-of-concept, making sure that the build could run after the merges.

Reply:
The mismatch on password/confirmation are more when the `user` is created, and incorrect password is during actual login, right? Want to make sure I'm matching where you see these are all applied.